Cargo.toml

```diff
@@ -17,6 +17,7 @@ bitflags = "2.5.0"
 bumpalo = { version = "3.16.0", features = ["collections", "boxed"] }
 dtoa = "1.0.9"
 base64 = "0.22.1"
+serde_json = "1.0.117"
```

Comment:
not sure if it's fine to add it here, but I thought that this would be an easy data format for accepting bindings